app/services/algolia/vacancy_search_builder.rb

```diff
@@ -84,7 +84,7 @@ def build_stats(page, pages, results_per_page, total_results)
   end
 
   def search
-    Vacancy.search(
+    Vacancy.includes(organisation_vacancies: [:organisation]).search(
```

Comment:
What does this do exactly?

Reply:
it eager-loads organisations, this will improve the performance

https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations